src/main/java/org/troyargonauts/Constants.java

```diff
@@ -6,5 +6,19 @@ public interface Manipulator {
         public static final int kForwardChannel = 0;
         public static final int kReverseChannel = 1;
     }
+    public interface Limelight {
+        public static final double LIMELIGHT_HEIGHT = 32.625;
```

Comment:
Public static final is redundant, all the interface requires in the datatype and variable name.
Remove the "public static final" from all variables inside the interface.

Reply:
Remove Manipulator Interface from this file.